tools/docker/builder/openwrt/Dockerfile

```diff
@@ -81,8 +81,9 @@ RUN echo "src-git prpl ${PRPL_FEED}" >> feeds.conf.default \
     && scripts/feeds install -a \
     && cp "/home/openwrt/prplmesh_config" .config \
     && make defconfig \
-    && make -j$(nproc) V=sc \
-    && make -j$(nproc) \
+    && cat .config \
+    && make package/feeds/prpl/prplmesh/compile -j$(nproc) V=sc \
+    && make package/feeds/prpl/prplmesh/compile -j$(nproc) \
     && make package/feeds/prpl/prplmesh/clean
     # For some reason, make has to be run twice because
```

Comment:
We previously had to run it twice because the build system would always try to rebuild the dependencies on the second run (but not on the third). I will check if it's still the case with this change.

Running make then clean is just a way to have the dependencies (but *not* prplmesh itself) already compiled inside the image. I don't know any way to build only the dependencies of a package in OpenWrt.

Reply:
Yes, running make twice is still needed.
We can probably avoid it by using CONFIG_AUTOREBUILD, but that can probably be done in another PR, because it needs testing.